coap-gateway/service/observation/observedResource.go

```diff
@@ -17,6 +21,7 @@ type Observation = interface {
 type observedResource struct {
 	href         string
 	resInterface string
+	etags        [][]byte
```

Comment:
I think that the `etag(s)` is never used (retrieved or updated) after the observation for given resource has been set up. Do we need to store it?

Reply:
fixed - removed from struct